documentation/2.0/content/rcuinfo.md

```diff
@@ -161,3 +182,205 @@ domainInfo:
+                    TestTableName: SQL ISVALID
+                JDBCDataSourceParams:
+                    GlobalTransactionsProtocol: None
+                    JNDIName: jdbc/AuditAppendDataSource
+                JDBCDriverParams:
+                    URL: --FIX ME--
+                    PasswordEncrypted: --FIX ME--
+                    DriverName: oracle.jdbc.OracleDriver
+                    Properties:
+                        user:
+                            Value: FMW1_IAU_APPEND
+        mds-owsm:
+            Target: cluster-1,admin-server
+            JdbcResource:
+                JDBCConnectionPoolParams:
+                    InitialCapacity: 0
+                    TestConnectionsOnReserve: true
+                    ConnectionCreationRetryFrequencySeconds: 10
+                    TestTableName: SQL ISVALID
+                    TestFrequencySeconds: 300
+                    SecondsToTrustAnIdlePoolConnection: 0
+                JDBCDataSourceParams:
+                    GlobalTransactionsProtocol: None
+                    JNDIName: jdbc/mds/owsm
+                JDBCDriverParams:
+                    URL: --FIX ME--
+                    PasswordEncrypted: --FIX ME--
+                    DriverName: oracle.jdbc.OracleDriver
+                    Properties:
+                        user:
+                            Value: FMW1_MDS
+                        oracle.net.CONNECT_TIMEOUT:
+                            Value: '120000'
+                        SendStreamAsBlob:
+                            Value: 'true'
+```
+
+
+#### Oracle Active Grid Link Data Source (AGL)
+
+For setting data source to access Oracle Active Grid Link database, besides updating the `URL`, `PasswordEncrypted`, and the `user` property value; 
+you can specify additional `JDBCOracleParams` under `JdbcResource` of each data source.  For example,
+
+```yaml
+      mds-owsm:
+            Target: cluster-1,admin-server
+            JdbcResource:
+              JDBCOracleParams:
+                OnsNodeList:  'node1, node2'
+                ....
+```
+
+You can run WDT tool `modelHelp.sh -oracle_home <oracle home> resources:/JDBCSystemResource/JdbcResource/JDBCOracleParmas` for the complete list 
+of fields.
+
+#### Multi Data Sources
+
+If your database does not support high availability natively such as RAC or AGL, then WebLogic provides a Multi Data Source option, which basically the main data source is a logical data source that
+contains multiple physical data sources.   In order to create a spare model,  you can for each data source
+
+1. Update the templated data source `URL`, `EncryptedPassword`, and 'user' property value.  In the `URL` part, you can specify one of the accessible physical database URL.
+2. Duplicate each datasource two times
+3. Change the duplicated data source name, `JNDIName` by appending `-1`, `-2` etc. The `-1`, `-2` data sources are the physical data source, you will need to update their `URL` too.
```

Comment:
etc -> , and such.
data source, you will -> data sources; (semi-colon) you will